### ISSUE TITLE
exec-path not set

### ISSUE THREAD

**Issue by vkazanov** [rejeep/nvm.el#16]
I use this library to setup nvm-related paths in my .emacs.

There's one issue though: it doesn't update the `exec-path` variable, which is used by the (start-process) function to launch Emacs subprocesses. This, in turn, is used by tide to launch the tsserver.

Currently I have hack doing something like:

```
  (nvm-use "v8.16.1")
  (add-to-list (getenv "NVM_BIN")  exec-path)
```

Easy but probably should in nvm.el itself. What do you think?

**Comment by rejeep** [comment]
That make sense to me 👍

**Comment by vkazanov** [comment]
Do you accept PRs or want to do things yourself? ;-)

**Comment by rejeep** [comment]
I don't do things myself, so a PR would be much welcome! 😉

**Comment by vkazanov** [comment]
It turns out the non-stable version from Melpa already does this..! I 

Maybe it's time for a new release? ;-)

**Comment by rejeep** [comment]
What do you know! 🙃 I released v0.3.0 now and pushed a new tag.

**Comment by vkazanov** [comment]
Thanks! I was really surprised to find `exec-path` checking in cloned repo tests! :laughing:
